coordinator/migrations/2024-03-18-083352_add_order_matching_fees_to_position/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE positions ADD COLUMN "order_matching_fees" BIGINT;
```

Comment:
❓ Can't we derive the order matching fee from the data on the position support resizing of already existing positions?

Reply:
I think you are right, working on it!